fiboa_cli/publish.py

```diff
@@ -190,7 +190,7 @@ def publish(dataset, directory, cache, source_coop_extension):
     if not os.environ.get("AWS_SECRET_ACCESS_KEY"):
         log(f"Get your credentials at {source_coop_url}manage/", "info")
         log("  Then press 'ACCESS DATA',\n  and click 'Create API Key',", "info")
-        log("  Run export AWS_DEFAULT_REGION=us-west-2 AWS_ACCESS_KEY_ID=<> AWS_SECRET_ACCESS_KEY=<>\n"
+        log("  Run export AWS_DEFAULT_REGION=https://data.source.coop AWS_ACCESS_KEY_ID=<> AWS_SECRET_ACCESS_KEY=<>\n"
```

Comment:
Shouldn't that be the endpoint URL instead of the region?

Reply:
Yes, thanks for checking